src/app/dashboard-widgets/create-work-item-widget/create-work-item-widget.component.html

```diff
@@ -1,44 +1,120 @@
+            <p class="f8-work-item-card-data-totals--link">
+              <a href="#" class="btn btn-link">View Planner</a>
+            </p>
+          </div>
+          <div class="f8-work-item-card-data-status">
+            <div class="f8-work-item-card-data-status--resolved">
+              <div class="f8-work-item-card-data-status--resolved_color">
+                <span class="pficon pficon-resources-full"></span>
+              </div>
+              <div class="f8-work-item-card-data-status--resolved_color">
+                102<!-- replace this with queried number -->
+              </div>
+              <div>
+                Resolved
+              </div>
+            </div>
+            <div class="f8-work-item-card-data-status--in-progress">
+              <div class="f8-work-item-card-data-status--in-progress_color">
+                <span class="pficon pficon-resources-almost-full"></span>
+              </div>
+              <div class="f8-work-item-card-data-status--in-progress_color">
+                175<!-- replace this with queried number -->
+              </div>
+              <div>
+                In progress
+              </div>
+            </div>
+            <div class="f8-work-item-card-data-status--open">
+              <div class="f8-work-item-card-data-status--open_color">
+                <span class="fa fa-fire"></span>
+              </div>
+              <div class="f8-work-item-card-data-status--open_color">
+                358<!-- replace this with queried number -->
+              </div>
+              <div>
+                Open
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="f8-work-item-card-chart"> <!-- replace this with actual D3 chart -->
+          <div class="f8-work-item-card-chart--resolved">
+          </div>
+          <div class="f8-work-item-card-chart--in-progress">
+          </div>
+          <div class="f8-work-item-card-chart--open">
+          </div>
+        </div>
+      </div>
     </div>
-    <h2 class="card-pf-title">
-      <a id="spacehome-my-workitems-title" [routerLink]="[contextPath | async, 'plan']">My Work Items</a>
-      <span id="spacehome-my-workitems-badge" class="badge f8-card-badge">{{myWorkItemsCount | async}}</span>
-    </h2>
   </div>
-  <div class="card-pf-body f8-card-body">
-    <div class="f8-blank-slate-card" *ngIf="(myWorkItemsCount | async) === 0">
-      <h3>You can view your recent work items</h3>
-      <p>
-        Planner will help you to create different types of work-items, allows you to assign team members and iterations.
-      </p>
-      <div class="f8-blank-slate-main-action">
-        <button id="spacehome-my-workitems-create-button" class="btn btn-primary btn-lg" [routerLink]="[{ outlets: { action: 'add-work-item' } }]">Create work item</button>
+  <div id="spacehome-my-workitems-card" class="create-work-item-widget card-pf f8-card" default-level>
```

Comment:
What does `default-level` do? I can find the reference for it. Though I can see it is used in several places.
Can you confirm it actually does what you think it does?

Reply:
`default-level` identifies the default view that a non-local (i.e. prod-preview or badger) user will see. `user-level` only shows the content that is contained within that selector.